### PR TITLE
Add message board in portfolio.

### PR DESCRIPTION
**Pull request by anitaHey**
Add two parts of content (education,work experence) and message board.

TODO: Three parts of my portfolio and RWD.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/data/Message.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+// Copyright 2019 Google LLC
```
Comment (louisc1028) [inline portfolio/src/main/java/com/google/sps/data/Message.java:1]: 2020

File: portfolio/src/main/java/com/google/sps/servlets/ListMessageServlet.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
+// Copyright 2019 Google LLC
```
Comment (louisc1028) [inline portfolio/src/main/java/com/google/sps/servlets/ListMessageServlet.java:1]: same here

File: portfolio/pom.xml [hunk 3]
```diff
@@ -35,7 +45,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
-          <deploy.projectId>YOUR_PROJECT_ID_HERE</deploy.projectId>
+          <deploy.projectId>AnitaChang-Portfolio</deploy.projectId>
```
Comment (louisc1028) [inline portfolio/pom.xml:48]: Shouldn't this be the project id that corresponding to the one in Google Cloud Platform?

Comment (anitaHey) [reply portfolio/pom.xml]: Oh yes, I forgot to push the latest version for this file. Thanks!

File: portfolio/src/main/java/com/google/sps/servlets/ListMessageServlet.java (hunk 2 above)
Comment (anitaHey) [reply portfolio/src/main/java/com/google/sps/servlets/ListMessageServlet.java]: Fix it. Thanks!

File: portfolio/src/main/java/com/google/sps/data/Message.java (hunk 1 above)
Comment (anitaHey) [reply portfolio/src/main/java/com/google/sps/data/Message.java]: Fix it. Thanks!

**Pull request merged**